backend/config.py

```
@@ -1,49 +1,79 @@
+            return {
+                'echo': False,  # Set to True for SQL debugging
+                'pool_pre_ping': True,  # Validate connections before use
+                'pool_recycle': 3600,  # Recycle connections every hour
+                'pool_size': 10,
+                'max_overflow': 20
+            }
+        else:
+            # SQLite and other databases
+            return {
+                'echo': False,  # Set to True for SQL debugging
+            }
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)